Enterprise-CMCS/macpro-mdct-mfp
# mdct-2729 report drawer and drawer report page and tests

### Description
Added ReportDrawer and DrawerReportPage. Adjusted the Report Page Wrapper to include the switch statement again to look at the pagetype. Also hardcoded entities to get the drawer to actually show up. This will need to get updated when other pagetypes get added, but it's fine for now.


### Related ticket(s)


MDCT-2729

---
### How to test
Go to /standard and you'll see the the drawer page and the ability to enter into a report drawer.


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
